src/back/controllers/api/routes/categories/categories.js

```diff
@@ -4,6 +4,14 @@ const categoryModel = require('./../../models/category');
 const userModel = require('./../../models/user');
 
 const getAll = (req, res, next) => {
+    categoryModel.find({}, (err, categories) => {
+        if (err) next(next);
```

Comment:
you should add else statement or return 
otherwise this will cause error: attempt to send response twice